contracts/truefi2/FixedTermLoanAgency.sol

```diff
@@ -92,6 +92,8 @@ contract FixedTermLoanAgency is IFixedTermLoanAgency, UpgradeableClaimable {
 
     IStakingVault public stakingVault;
 
+    mapping(address => IFixedTermLoan[]) public borrowerLoans;
```

Comment:
I just remembered that there's a max of one FTL per borrower, so all we would need to track is the single FTL that a borrower might have.

I wonder if there's a standard Bidirectional map implementation for Solidity...